huaweicloud/services/acceptance/iotda/resource_huaweicloud_iotda_batchtask_test.go

```diff
@@ -0,0 +1,291 @@
+package iotda
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/huaweicloud/huaweicloud-sdk-go-v3/services/iotda/v5/model"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+)
+
+func getBatchTaskResourceFunc(conf *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := conf.HcIoTdaV5Client(acceptance.HW_REGION_NAME, withDerivedAuth())
+	if err != nil {
+		return nil, fmt.Errorf("error creating IoTDA v5 client: %s", err)
+	}
+
+	// There is no need to handle pagination related parameters here, as it only applies to the structure of subtasks.
+	resp, err := client.ShowBatchTask(&model.ShowBatchTaskRequest{
+		TaskId: state.Primary.ID,
+	})
+
+	if err != nil {
+		return nil, fmt.Errorf("error querying IoTDA batch task: %s", err)
+	}
+
+	return resp, nil
+}
+
+func TestAccBatchTask_withTargetsField(t *testing.T) {
```

Comment:
we must hava an acceptance with **basic** suffix.
```suggestion
func TestAccBatchTask_basic(t *testing.T) {
```

Reply:
fixed.